app/assets/javascripts/actions/projectBoard.js

```diff
@@ -24,7 +23,7 @@ const receiveProject = data => ({
 });
 
 export const fetchProjectBoard = projectId => {
-  return dispatch => {
+  return (dispatch , getState, { ProjectBoard }) => {
```

Comment:
remove space before first comma :nail_care:

Reply:
Hey @kostadriano, the space is still there 😛